@jaykayudo  Dropped you a message in google chat. Basically, I did try to install using the contributing guide, unfortunately I am getting many errors that does not seem related to the changes I did. For example:
error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:135:14
        args.append_serde(from);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:136:14
        args.append_serde(to);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:137:14
        args.append_serde(token_id);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:153:14
        args.append_serde(from);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:154:14
        args.append_serde(to);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:155:14
        args.append_serde(token_id);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:156:14
        args.append_serde(data);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:169:14
        args.append_serde(interface_id);
             ^^^^^^^^^^^^

error[E0002]: Method `unwrap_and_cast` not found on type `core::result::Result::<core::array::Span::<core::felt252>, core::array::Array::<core::felt252>>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721.cairo:172:14
            .unwrap_and_cast()
             ^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:4:5
use openzeppelin_utils::UnwrapAndCast;
    ^^^^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:5:5
use openzeppelin_utils::selectors;
    ^^^^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:6:5
use openzeppelin_utils::serde::SerializedAppend;
    ^^^^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:7:5
use openzeppelin_utils::try_selector_with_fallback;
    ^^^^^^^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:34:14
        args.append_serde(operator);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:35:14
        args.append_serde(from);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:36:14
        args.append_serde(token_id);
             ^^^^^^^^^^^^

error[E0002]: Method `append_serde` not found on type `core::array::Array::<?0>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:37:14
        args.append_serde(data);
             ^^^^^^^^^^^^

error[E0002]: Method `unwrap_and_cast` not found on type `<missing>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/erc721/dual721_receiver.cairo:45:14
            .unwrap_and_cast()
             ^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:6:9
    use openzeppelin_utils::cryptography::nonces::NoncesComponent;
        ^^^^^^^^^^^^^^^^^^

error[E0006]: Identifier not found.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:7:9
    use openzeppelin_utils::cryptography::snip12::SNIP12Metadata;
        ^^^^^^^^^^^^^^^^^^

error: Trait has no implementation in context: openzeppelin_token::erc20::erc20::ERC20Component::ERC20HooksTrait::<openzeppelin_token::tests::mocks::erc20_votes_mocks::DualCaseERC20VotesMock::ContractState>.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:21:43
    impl ERC20MixinImpl = ERC20Component::ERC20MixinImpl<ContractState>;
                                          ^^^^^^^^^^^^^^

error: Trait has no implementation in context: openzeppelin_token::erc20::erc20::ERC20Component::ERC20HooksTrait::<openzeppelin_token::tests::mocks::erc20_votes_mocks::DualCaseERC20VotesMock::ContractState>.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:22:41
    impl InternalImpl = ERC20Component::InternalImpl<ContractState>;
                                        ^^^^^^^^^^^^

error[E0002]: Method `transfer_voting_units` not found on type `openzeppelin_token::erc20::extensions::erc20_votes::ERC20VotesComponent::ComponentState::<TContractState>`. Did you import the correct trait and impl?
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:77:35
            erc20_votes_component.transfer_voting_units(from, recipient, amount);
                                  ^^^^^^^^^^^^^^^^^^^^^

error: Trait has no implementation in context: openzeppelin_token::erc20::erc20::ERC20Component::ERC20HooksTrait::<openzeppelin_token::tests::mocks::erc20_votes_mocks::DualCaseERC20VotesMock::ContractState>.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:91:20
        self.erc20.initializer(name, symbol);
                   ^^^^^^^^^^^

error: Type annotations needed. Failed to infer ?2.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:28:5
    #[storage]
    ^^^^^^^^^^

error: Invalid drop trait implementation, Candidate impl core::traits::SnapshotDrop::<?0> has an unused generic parameter.
 --> /home/rustdev/.cache/scarb/registry/src/scarbs.xyz-9djtpev4jug5q/openzeppelin_token-0.17.0/src/tests/mocks/erc20_votes_mocks.cairo:28:5-36:5
      #[storage]
 _____^
| ...
|     }
|_____^

But the number is really large, so I am not able to isolate the tests on the changes I did, I just push another commit, normalizing the code to cairo 1.x., hope it works...

--- a/src/tests/test_utils.cairo
+++ b/src/tests/test_utils.cairo
@@ -1,6 +1,6 @@
+use core::integer::u256;
 use snforge_std::{ContractClassTrait, DeclareResultTrait, declare};
 use starknet::{ContractAddress, contract_address_const};
-use core::integer::u256;
 use crate::components::logics::compute_admin_fee;
 
 pub fn deploy_contract() -> (ContractAddress, ContractAddress) {

